### PR TITLE
Document how to cross compile chromeos

### PR DESCRIPTION
**Pull request by petermcneeleychromium**
I wanted to document how I got chromeos to build and run amber.

There is probably a better way to do this directly in cmake

### CODE REVIEW

**Comment by petermcneeleychromium** [comment]
There are probably many ways of doing this more "correctly"
https://github.com/SX-Aurora/CMake-toolchain-file

File: README.md [hunk 1]
```diff
@@ -171,6 +171,84 @@ cd /data/local/tmp
...
+ninja amber
+# Copy over the amber binary to the DUT (note the root disk partion has limited space)
+scp amber device:/root/amber
+# An example of how to copy over some amber scripts
+scp ../../tests/cases/* device:/root/
+
+```
+Now by ssh-ing into the DUT you can locally run any amber script. Also, vulkan layers
+may not be available to this executable, so make sure to use the `-d` flag to disable
+Vulkan layers:
```
Comment (dj2) [inline README.md:250]:
```suggestion
Vulkan layers.
```

File: tests/benchmarks/parallel_advantage/parallel_cos_2x.amber [hunk 2]
```diff
@@ -0,0 +1,83 @@
+#!amber
```
Comment (dj2) [inline tests/benchmarks/parallel_advantage/parallel_cos_2x.amber:1]: Did you mean to add these amber script files?

Comment (petermcneeleychromium) [reply tests/benchmarks/parallel_advantage/parallel_cos_2x.amber:1]: not in this pr :)

**Comment by petermcneeleychromium** [comment]
We can discuss in person if you actually want this in the readme.

**Pull request merged**